docs/compilers/CSharp/testplans/stackalloc_initializers.md

```diff
@@ -10,24 +10,24 @@ In such case we can just check them off.
 ### correctness ###
 - [ ] check that partial result is not visible. - if element throws and exception caught, whole thing is not reassigned to the new value. (try spans and ordinary stackallocs)
 - [ ] if we refer to the elements of outer local in the initializers we still see the old values.
-- [ ] put `await` in the size or in the middle of element initializers
+- [x] put `await` in the size or in the middle of element initializers
```

Comment:
I think we might want to just delete the file now - to not duplicate where the tracking happens - since we have an issue for this.

Reply:
It seemed a good idea at a time, but not anymore. Editing a file is not so easy - especially if it needs multiple small edits and by multiple people.

---
In reply to: [171699064](https://github.com/dotnet/roslyn/pull/25101#discussion_r171699064) [](ancestors = 171699064)